sda/internal/helper/helper.go

```diff
@@ -153,6 +154,50 @@ func CreateRSAkeys(prPath, pubPath string) error {
 	return nil
 }
 
+func CreateSSHKey(path string) error {
+	privatekey, err := rsa.GenerateKey(rand.Reader, 2048)
+	if err != nil {
+		return err
+	}
+	pk := &privatekey.PublicKey
+
+	// dump private key to file
+	privateKeyBytes := x509.MarshalPKCS1PrivateKey(privatekey)
+	privateKeyBlock := &pem.Block{
+		Type:  "RSA PRIVATE KEY",
+		Bytes: privateKeyBytes,
+	}
+	encPrivateKeyBlock, err := x509.EncryptPEMBlock(rand.Reader, privateKeyBlock.Type, privateKeyBlock.Bytes, []byte("password"), x509.PEMCipherAES256) //nolint:staticcheck
+	if err != nil {
+		return err
+	}
+	privatePem, err := os.Create(path + "/id_rsa")
+	if err != nil {
+		return err
+	}
+	err = pem.Encode(privatePem, encPrivateKeyBlock)
+	if err != nil {
+		return err
+	}
+
+	err = os.Chmod(path+"/id_rsa", 0600)
+	if err != nil {
+		return err
+	}
+
+	publicKey, err := ssh.NewPublicKey(pk)
+	if err != nil {
+		return err
+	}
+	pubKeyBytes := ssh.MarshalAuthorizedKey(publicKey)
+	err = os.WriteFile(path+"/id_rsa.pub", pubKeyBytes, 0600)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
```

Comment:
It seems this function is only called by `storage_test.go`, so better move it there? Especially since x509.EncryptPEMBlock is deprecated and therefore we wouldn't want to actually use this function anywhere else.